contrib/syntax/nano/README.md

```diff
@@ -0,0 +1,18 @@
+Dockerfile.nanorc
+=================
+
+Dockerfile syntax highlighting for nano
+
+Installation
+------------
+1. Create a `nano` directory in `/usr/local/share/`
+ * `mkdir /usr/local/share/nano`
+
+2. Copy `Dockerfile.nanorc` to` /usr/local/share/nano`
+ * `cp Dockerfile.nanorc /usr/local/share/nano/`
```

Comment:
Since the path chosen here has to be added to `~/.nanorc` anyhow, wouldn't it be better to recommend that users put this into `~/.nano/syntax` (which seems to be a common place people recommend extensions to nano's syntax highlighting be installed per-user)?